add-ons/shadow/cmd/main/ss_controller.go

```diff
@@ -30,12 +30,10 @@ import (
 
 const (
 	// DefaultMeshServer is the default access address of the easemesh control plane
-	DefaultMeshServer = "easemesh-controlplane-svc.easemesh:2381"
+	DefaultMeshServer = "easemesh-control-plane-service.easemesh:2381"
```

Comment:
Is this change backward compatible?

Reply:
Nope, along with the incompatible change of https://github.com/megaease/easegress/pull/528 that I discussed with EaseAgent team last week. So I just do it in one shot for more consistency.